bakery-src/scripts/copy_resources_s3.py

```diff
@@ -242,18 +272,22 @@ async def check_s3_existence_worker(resource):
     print('Time it took to check: {}s'.format(elapsed))
 
     # Upload to s3
-    print('Uploading to S3 ', end='', flush=True)
+    chunk_size = max(
+        len(upload_resources) * 2 // MAX_THREAD_UPLOAD_S3,
```

Comment:
I multiply by 2 here because `upload_arg_gen` yields 2 items per resource (metadata and resource), effectively doubling the length of the list. Honestly, the chunk sizes I picked are very arbitrary. We may be able to fine tune them more for better performance.

Reply:
Looks good to me.